### ISSUE TITLE
Update User Roles

### ISSUE THREAD

**Issue by theGaryLarson** [Computing-For-All/nextjs-issue-tracker#202]
Admin needs to have the capability to assign roles.

Process: 

Account is created ADMIN role receives email of new account.
Navigate to ADMIN page 
Open section that shows users (should be filterable by role).
Have an option to CRUD roles.
Submit changes.

**Comment by not-damien** [comment]
resolved with pr: https://dev.azure.com/CFA1/Career%20Services/_git/CoalitionWebsite/pullrequest/249

**Comment by rorosaurus** [comment]
Verified changing user roles. Looks good! Is this `NOT CONNECTED TO BACKEND YET` text still needed?

**Comment by not-damien** [comment]
Yeah that can be removed, its fully connected now and i forgot to remove it
